containers/server-bundle/Dockerfile

```diff
@@ -24,26 +28,26 @@ ENV ZWED_agent_http_port='8542'
 
 ENV LAUNCH_COMPONENT_GROUPS=DESKTOP,GATEWAY
 
-RUN mkdir /root/zowe/utils && \
-    mkdir /root/zowe/samples
-COPY utils/ /root/zowe/utils/
-COPY start.sh /root/zowe/samples/
+RUN mkdir /home/zowe/utils && \
+    mkdir /home/zowe/samples
+COPY utils/ /home/zowe/utils/
+COPY start.sh /home/zowe/samples/
```

Comment:
its because unlike with a product, we dont have a way to deliver a bundle of software, so we hide the samples inside the image and have documentation to explain how to copy them out of the image. the start.sh is to make it very easy to start the docker container if you are new to docker.